docs/python/environments.md

```diff
@@ -16,20 +16,22 @@ To select a specific interpreter, select the **Python: Select Interpreter** comm
 
 ![Python: Select Interpreter command](images/environments/select-interpreters-command.png)
 
-This command automatically looks for and displays a list of available Python interpreters.
+This command automatically looks for and displays a list of available Python interpreters, as well as Conda and virtual environments:
 
 ![List of interpreters](images/environments/interpreters-list.png)
 
-Selecting an interpreter from the list configures your User [Settings](/docs/getstarted/settings.md) accordingly. The Status Bar shows the current interpreter and when selected brings up a list of available interpreters. The Status Bar also reflects when no interpreter is selected.
-
-![No interpreter selected](images/environments/no-interpreter-selected-statusbar.png)
-
 The extension looks for interpreters in the following locations:
 
 - Standard paths such as `/usr/local/bin`, `/usr/sbin`, `/sbin`, `c:\\python27`, `c:\\python36`, etc.
-- Virtual environments located under the workspace (project) folder
+- Virtual environments located under the workspace (project) folder, unless the `python.terminal.activateEnvironments` setting is `false`, in which case virtual environments are ignored.
```

Comment:
This actually didn't change. `activateEnvironments` only controls *activating* an environment, not discovering them.